src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -112,20 +116,69 @@ export const ARShowScreen = ({ navigation, route }) => {
   );
 };
 
+/**
+ * index creation function for model and texture
+ *
+ * @param {string} startDate           start date of the model
+ * @param {number} timePeriodInDays    time period in days of the model
+ * @param {array}  scenes              array of models
+ *
+ * @return {object} both parsed values as an object, like { modalIndex: 1, textureIndex: 1 }
+ */
+const multipleSceneIndexGenerator = async ({ startDate, timePeriodInDays, scenes }) =>
+  new Promise((resolve) => {
+    let modalIndex, textureIndex;
```

Comment:
do you mean "model" with "modal"? a modal is like a window/popup in tech speech: https://en.wikipedia.org/wiki/Modal_window

Reply:
Yeah, I'll change that. 👍